### PR TITLE
Update do.m3u

### PR DESCRIPTION
**Pull request by sguinetti**
This PR adds AkíTV from https://akitv.net/

### CODE REVIEW

File: channels/do.m3u [hunk 1]
```diff
@@ -3,6 +3,8 @@
 https://ss6.domint.net:3028/ame_str/amecanal47/playlist.m3u8
 #EXTINF:-1 tvg-id="AnimeZoneTV.do" tvg-country="LATAM;PT;ES" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/AnimeZoneXTV/picture?width=300&height=300" group-title="Animation",Anime Zone TV (480p) [Not 24/7]
 http://azxtv.com/hls/stream.m3u8
+#EXTINF:-1 tvg-id="AkíTV.do" tvg-country="DO" tvg-language="" tvg-logo="" group-title="",AkíTV
+https://cm.hostlagarto.com:4445/Akitv/Akitv.myStream/playlist.m3u8
```
Comment (jkemp814) [inline channels/do.m3u:7]: Can't view it from here. Maybe others can.
https://cm.hostlagarto.com:4445/Akitv/Akitv.myStream/playlist.m3u8: Server returned 404 Not Found

Comment (LaneSh4d0w) [reply channels/do.m3u:7]: Might be not 24/7.

Comment (sguinetti) [reply channels/do.m3u:7]: I see this channel hours ago. So this channel is tagged Not 24/7.

File: channels/do.m3u [hunk 2]
```diff
@@ -3,7 +3,7 @@
 https://ss6.domint.net:3028/ame_str/amecanal47/playlist.m3u8
 #EXTINF:-1 tvg-id="AnimeZoneTV.do" tvg-country="LATAM;PT;ES" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/AnimeZoneXTV/picture?width=300&height=300" group-title="Animation",Anime Zone TV (480p) [Not 24/7]
 http://azxtv.com/hls/stream.m3u8
-#EXTINF:-1 tvg-id="AkíTV.do" tvg-country="DO" tvg-language="" tvg-logo="" group-title="",AkíTV
+#EXTINF:-1 tvg-id="AkíTV.do" tvg-country="DO" tvg-language="" tvg-logo="" group-title="",AkíTV [Not 24/7]
 https://cm.hostlagarto.com:4445/Akitv/Akitv.myStream/playlist.m3u8
```
Comment (jkemp814) [inline channels/do.m3u:7]: Working now

**Review by jkemp814 — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**